srv/salt/_modules/packagemanager.py

```diff
@@ -177,6 +177,7 @@ def __init__(self, **kwargs):
     def _refresh(self):
         """
         Refresh Zypper before updating
+        Raise RefreshFailed when returncode != 1
```

Comment:
Did you mean when returncode != 0 ?

Reply:
I don't even'raise' anymore; deleting the comment